src/ripple/rpc/tests/AccountInfo_test.cpp

```diff
@@ -0,0 +1,162 @@
+        }
+    }
+
+   // Test the "signer_lists" argument in account_info.
+   void testSignerLists()
+    {
+        using namespace jtx;
+        Env env(*this, features(featureMultiSign));
+        Account const alice {"alice"};
+        env.fund(XRP(1000), alice);
+
+        auto const withoutSigners = std::string ("{ ") +
+            "\"account\": \"" + alice.human() + "\"}";
+
+        auto const withSigners = std::string ("{ ") +
+            "\"account\": \"" + alice.human() + "\", " +
+            "\"signer_lists\": true }";
+
+        // Alice has no SignerList yet.
+        {
+            // account_info without the "signer_lists" argument.
+            auto const info = env.rpc ("json", "account_info", withoutSigners);
+            expect (! info[jss::result][jss::account_data].
+                isMember (jss::signer_lists));
+        }
+        {
+            // account_info with the "signer_lists" argument.
+            auto const info = env.rpc ("json", "account_info", withSigners);
+            auto const& data = info[jss::result][jss::account_data];
+            expect (data.isMember (jss::signer_lists));
+            auto const& signerLists = data[jss::signer_lists];
+            expect (signerLists.isArray());
+            expect (signerLists.size() == 0);
+        }
+
+        // Give alice a SignerList.
+        Account const bogie {"bogie"};
+
+        Json::Value const smallSigners = signers(alice, 2, { { bogie, 3 } });
+        env(smallSigners);
+        {
+            // account_info without the "signer_lists" argument.
+            auto const info = env.rpc ("json", "account_info", withoutSigners);
+            expect (! info[jss::result][jss::account_data].
+                isMember (jss::signer_lists));
+        }
+        {
+            // account_info with the "signer_lists" argument.
+            auto const info = env.rpc ("json", "account_info", withSigners);
+            auto const& data = info[jss::result][jss::account_data];
+            expect (data.isMember (jss::signer_lists));
+            auto const& signerLists = data[jss::signer_lists];
+            expect (signerLists.isArray());
+            expect (signerLists.size() == 1);
+            auto const& signers = signerLists[0u];
+            expect (signers.isObject());
+            expect (signers[sfSignerQuorum.jsonName] == 2);
+            auto const& signerEntries = signers[sfSignerEntries.jsonName];
+            expect (signerEntries.size() == 1);
+            auto const& entry0 = signerEntries[0u][sfSignerEntry.jsonName];
+            expect (entry0[sfSignerWeight.jsonName] == 3);
+        }
+
+        // Give alice a big signer list
+        Account const demon {"demon"};
+        Account const ghost {"ghost"};
+        Account const haunt {"haunt"};
+        Account const jinni {"jinni"};
+        Account const phase {"phase"};
+        Account const shade {"shade"};
+        Account const spook {"spook"};
```

Comment:
Thanks.  I stole these names from the MultiSign unit tests (https://github.com/ripple/rippled/blob/develop/src/ripple/app/tests/MultiSign.test.cpp).  I use these names for phantom accounts, so you probably don't need them in your pretty printer.

There are a couple more names in the MultiSign unit tests that are not accounted for in your list: "becky" and "cheri".  Other than that the 4-letter names I only use for regular keys.  So, again, they should not show up as master keys.  I hope that helps.